packages/connectivity_plus/connectivity_plus/example/integration_test/connectivity_plus_test.dart

```diff
@@ -38,5 +38,12 @@ void main() {
 
       expect(result, ConnectivityResult.ethernet);
     }, skip: !Platform.isMacOS);
+    
+    testWidgets('connectivity on Linux should be ethernet',
+        (WidgetTester tester) async {
+      final result = await _connectivity.checkConnectivity();
+
```

Comment:
need to remove a tab from here, waiting for review comments first so we do not run the CI too many times